crates/solidity/testing/perf/npm/src/main.mts

```diff
@@ -0,0 +1,83 @@
+import { SlangBindingsFileRunner, SlangBindingsProjectRunner } from "./slang.runner.mjs";
+import path from "node:path";
+import { SolcRunner } from "./solc.runner.mjs";
+import { checkCI, Runner, round2, Timing } from "./common.mjs";
+import commandLineArgs from "command-line-args";
+import commandLineUsage from "command-line-usage";
+import { SolidityParserRunner } from "./solidity.parser.runner.mjs";
```

Comment:
nit: I suggest moving the runners to a dedicated `runners` folder, for discovery:

- `crates/solidity/testing/perf/npm/src/runners/antlr.mts`
- `crates/solidity/testing/perf/npm/src/runners/slang.mts`
- `crates/solidity/testing/perf/npm/src/runners/solc.mts`